pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -448,10 +483,7 @@ let
       license = licenses.asl20;
       maintainers = with maintainers; [ abbradar ];
       platforms = with platforms; linux ++ darwin;
-      # More vulnerabilities in 2.11.1 really; https://github.com/tensorflow/tensorflow/releases
-      knownVulnerabilities = [ "CVE-2023-33976" ];
-      broken = true || # most likely needs dealing with protobuf/abseil updates
-        !(xlaSupport -> cudaSupport) || python.pythonVersion == "3.11";
+      broken = !(xlaSupport -> cudaSupport) || python.pythonVersion == "3.11";
```

Comment:
Isn't tensorflow 2.13 compatible with python3.11 ?

Reply:
Yes, upstream has enabled it, although I'm not sure about the status. I applied your corresponding changes from https://github.com/NixOS/nixpkgs/commit/0ee6d5912941dc5648759910e545516e3fb59121.